### PR TITLE
Section Part done

### PR DESCRIPTION
**Pull request by Smi-lin**

### CODE REVIEW

**Comment by DevBigEazi** [comment]
pls check your design very well and make sure it is responsive on all dvices

File: src/sections/Services.jsx [hunk 1]
```diff
@@ -1,5 +1,33 @@
+import { images } from "../constants";
+
 const Services = () => {
-  return <section>Services</section>;
+  return (
+    <section className="bg-[#EAF206] lg:h-[55vh] h-auto">
```
Comment (DevBigEazi) [inline src/sections/Services.jsx:5]: Pls leverage Tailwind configuration for colors...

**Review by DevBigEazi — CHANGES_REQUESTED** [review]
Don't use rem. Use Tailwind predefined values more. Check other components and let your code pattern be consistent with them.

**Comment by Smi-lin** [comment]
> Don't use rem. Use Tailwind predefined values more. Check other components and let your code pattern be consistent with them.

ok

**Review by Smi-lin — COMMENTED** [review]
I have changed all the rems to tailwind values

**Comment by Smi-lin** [comment]
Done

**Pull request merged**